include/envoy/upstream/load_balancer.h

```diff
@@ -10,6 +10,8 @@
 namespace Envoy {
 namespace Upstream {
 
+typedef std::vector<uint32_t> PriorityLoad;
```

Comment:
Can you add a comment describing this? The notion of "priority load" is introduced below, but there is no explanation of what the meaning of the numbers in this vector are.